src/arfsdao.ts

```diff
@@ -197,7 +202,7 @@ export class ArFSDAOAnonymous extends ArFSDAOType {
 				const { node } = edge;
 				cursor = edge.cursor;
 				const fileBuilder = await ArFSPublicFileBuilder.fromArweaveNode(node, this.arweave);
-				return fileBuilder.build(node);
+				return fileBuilder.build();
```

Comment:
Why did you get rid of the node data? Without the node data, it will cause a whole new wasted call for GQL data :(